volunteer.markdown

```diff
@@ -13,24 +13,27 @@ layout: page
 ## Get in touch and talk to us
 Join the conversation on [Slack](http://slack.hotosm.org/)! A large number of volunteers and staff discuss mapping, emergencies, Working Groups and other matters on our Slack channel.
 
-## Mapping Volunteers
+<span class="anchor" id="mapping-volunteers"></span>
```

Comment:
Please move HTML to to https://github.com/hotosm/hotosm-website/blob/gh-pages/_layouts/get-involved.html#L18-L20.